rpc/api.proto

```diff
@@ -155,6 +156,14 @@ message BalanceResponse {
 	int64 immature_reward = 3;
 }
 
+message ReceivedForAddressRequest {
+	string address = 1;
+	int32 required_confirmations = 2;
+}
+message ReceivedForAddressResponse {
+	float total = 1;
```

Comment:
as far as code goes, this should be a int64 (counted in satoshis).  In the rpc handler, any btcutil.Amount can simply by type converted to int64.

```
Total: int64(totalReceived),
```